internal/compiler/builtins.slint

```diff
@@ -152,6 +152,7 @@ component WindowItem {
     in property <brush> color <=> background;
     in property <string> title: "Slint Window";
     in property <bool> no-frame;
+    in property <bool> always-on-top;
```

Comment:
Is the ordering arbitrary, here? In the docs, the ordering is alphabetical.

Reply:
Yeah. Even the alphabetical ordering in the docs I would personally rather group by relevance. This looks good to me.